src/pages/index.tsx

```diff
@@ -1,32 +1,75 @@
 import React from 'react';
```

Comment:
I think it would be great to refer to the static assets with

```js
import DocusaurusImageUrl from '@site/static/img/docusaurus.png';

<img src={DocusaurusImageUrl} />;
```

Or at least please use `import useBaseUrl from '@docusaurus/useBaseUrl';`.

Read more here: https://docusaurus.io/docs/static-assets#in-jsx

Please apply this comment for all JSX/TSX files.